recipe-server/parquet_schemas/normandy.admin.docker.app/request.summary.cfg

```diff
@@ -0,0 +1,37 @@
+-- vim:ft=lua
+message_matcher = message_matcher .. " && Fields[Type] == 'request.summary'"
+
+s3_path_dimensions = {
+	{name="log", source="Type"},
+	{name="type", source="Fields[Type]"},
+	{name="date", source="Fields[Date]"},
+	{name="hour", source="Fields[Hour]"},
+}
+
+parquet_schema = [=[
+message Log {
+    required int64 Timestamp;
+    optional binary Type (UTF8);
+    optional binary Hostname (UTF8);
+    optional binary Logger (UTF8);
+    optional binary EnvVersion (UTF8);
+    optional int64 Severity;
+    optional int64 Pid;
+    optional binary fields_lang (UTF8);
+    optional binary fields_method (UTF8);
+    optional binary fields_path (UTF8);
+    optional binary fields_rid (UTF8);
+    optional binary fields_uid (UTF8);
+    optional binary fields_user_agent_browser (UTF8);
+    optional binary fields_user_agent_os (UTF8);
+    optional binary fields_user_agent_version (UTF8);
```

Comment:
The logs I see just have an `agent` field, unparsed. Can we include that as well as the parsed versions? (Or just that).

Also, where is the code that extracts these values from the mozlog format? (I know we talked about it living elsewhere, but  I'd like to see it).